### PR TITLE
feat(SystemAdvisor): ADVISOR-2783 - Add first impacted column

### PR DESCRIPTION
**Pull request by tompsota**
Resolves https://issues.redhat.com/browse/ADVISOR-2783

### CODE REVIEW

**Review by gkarat — CHANGES_REQUESTED** [review]
The code looks clean! ✅ There is a couple of things to improve:

After the slack discussion, it has turned out that we need to place the column after the "Modified" one. So please change it here in your PR.

Another thing, looks like a bug: I can't sort by "First impacted" in both directions (asc, desc):
1. Open the system details page.
2. Click on the First impacted header.
3. Click again in order to sort by the descending order.

It is expected that after the first click the table will be sorted in the ascending order by First impacted timestamp. But it is not doing that, however, for the desc order it works OK.

**Comment by gkarat** [comment]
@tompsota, ok, I would vote for a separate issue. Let's change only the order of the columns here and we are good to go

File: cypress/support/component.js
Comment (tompsota) [reply cypress/support/component.js:29]: That looks good! Should we wait until your PR is merged and continue with this PR afterwards?

Comment (gkarat) [reply cypress/support/component.js:29]: @bastilian, cool stuff, I think I could try to replace it in https://github.com/RedHatInsights/insights-advisor-frontend/pull/1041. I will work on it after we have this PR merged.

File: cypress/fixtures/systemRecommendations.json
Comment (gkarat) [inline cypress/fixtures/systemRecommendations.json:46]: Can we write one more test to check that the table translates and renders the timestamp correctly? You can just choose one row for this check.

File: package.json
Comment (gkarat) [inline package.json:28]: Let's make it consistent with other apps that already introduced this script:
```suggestion
    "test:openct": "cypress open --component",
```

**Review by gkarat — DISMISSED** [review]
Looks good to me. Just let's add one more test, and we are ready to merge it.

@adonispuente, @bastilian, I just wanted to explicitly point out to the change in `entityDetails` selector: we decided to move it out of the `BaseSystemAdvisor` to the wrapper. This lets test the component way easier since `entityDetails` is added dynamically by Inventory => hard to make a proper workaround in tests.

File: cypress/support/component.js
Comment (bastilian) [reply cypress/support/component.js:29]: @gkarat 👍 I think it might even prevent a bit of future headache, if things change in what scalprum queries from the global namespace.

File: cypress/fixtures/systemRecommendations.json
Comment (tompsota) [reply cypress/fixtures/systemRecommendations.json:46]: Let me know if this is what you expected.. Other, simpler solution would be to only check if the timestamp value contains 'ago' and maybe that would be sufficient as well

**Review by gkarat — APPROVED** [review]

**Pull request merged**
